opensir/models/model.py

```diff
@@ -43,12 +43,18 @@ def __init__(self):
     class InvalidParameterError(Exception):
         """Raised when an initial parameter of a value is not correct"""
 
-        pass
-
     class InvalidNumberOfParametersError(Exception):
         """Raised when the number of initial parameters is not correct"""
 
-        pass
+    class InconsistentDimensionsError(Exception):
+        """Raised when the length of the days array is
+        not equal to the dimension of the observed cases, or if the length
+        of fit_index has a length different than the length
+        of the parameter array self.p"""
+
+    class InvalidInputsForFittingError(Exception):
```

Comment:
I would drop this one, since it's just a specific case of "InvalidParameterError".
If more details are needed, it's always possible to raise with error messages:
```python
raise InvalidNumberOfError("Length of the days array is not equal to the dimension of the observed cases")
```

Reply:
Good point, will implement soon